frontend/src/initialize.css

```
@@ -0,0 +1,472 @@
+    overflow: auto;
+}
+
+/*
+   * 1. Correct inheritance and scaling of font-size for preformatted text.
+   * 2. Address odd `em`-unit font size rendering in all browsers.
+   */
+code,
+kbd,
+pre,
+samp {
+    font-family: monospace, monospace;
```

Biome (2.1.2)

[error] Duplicate font names are redundant and unnecessary: monospace

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)